src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1502,6 +1502,51 @@ public static IEnumerable<string> ParseConditionalCompilationSymbols(string valu
             return defines.AsEnumerable();
         }
 
+        public static bool TryParseLanguageVersion(string value, out LanguageVersion version)
```

Comment:
I don't think it's a big problem given that you're in a feature branch, but I'm confused, I thought you were going to wait for the change to go through master first.